tests/spec/core/issues-notes-spec.js

```diff
@@ -84,14 +83,13 @@ describe("Core — Issues and Notes", () => {
         "<p class='ednote' title='EDNOTE-TIT'>EDNOTE</p>",
     };
     const doc = await makeRSDoc(ops);
-    const $not = $("div.ednote", doc);
-    const $pnot = $not.find("p");
-    expect($not.find("div.ednote-title").length).toEqual(1);
-    expect($not.find("div.ednote-title").text()).toEqual(
+    const edNote = doc.querySelector("div.ednote");
+    expect(edNote.querySelectorAll("div.ednote-title").length).toBe(1);
+    expect(edNote.querySelector("div.ednote-title").textContent).toBe(
       "Editor's note: EDNOTE-TIT"
     );
-    expect($pnot.attr("title")).toBeUndefined();
-    expect($pnot.text()).toEqual("EDNOTE");
+    expect(edNote.querySelector("p").getAttribute("title")).toBeNull();
+    expect(edNote.querySelector("p").textContent).toBe("EDNOTE");
```

Comment:
Do we have a specific reason to remove variables and prefer inlining? Personally I would prefer the reverse way...

Reply:
No specific reason. Should I rewrite them?

Reply:
Repeated inline code is bad for readability, so yes for where the original had variables. Probably okay for where the original didn't, to ease reviewing 😁

Reply:
Gotcha!